setup.cfg

```diff
@@ -46,7 +46,6 @@ install_requires =
     click-loglevel ~= 0.2
     in_place ~= 0.4
     pydantic ~= 1.7
-    PyGithub ~= 1.55
```

Comment:
I wonder why you decided to rewrite those interfaces without using pygithub?

Reply:
Pygithub doesn't support fetching workflow run for a given commit, nor does it support several other run-related things, nor does it properly parse timestamps, and retrying requests via Pygithub when rate limits are hit was harder than it needed to be.  I think tinuous has reached the point where Pygithub is more of a hindrance than a help.